### PR TITLE
feat(rpc-testing-utils): support test block for trace_call_many

### PR DESCRIPTION
**Pull request by DoTheBestToGetTheBest**
Do you agree with these CallRequest data field  for these test blocks?

### CODE REVIEW

File: crates/rpc/rpc-testing-util/src/trace.rs [hunk 1]
```diff
@@ -319,4 +320,29 @@ mod tests {
         println!("Total successes: {}", successes);
         println!("Total failures: {}", failures);
     }
+
+    #[tokio::test]
+    async fn can_create_trace_call_many_stream() {
+        let client = HttpClientBuilder::default().build("http://localhost:8545").unwrap();
+
+        let call_request_1 = CallRequest { from: None, to: None, gas: None, ..Default::default() };
+        let call_request_2 = CallRequest { from: None, to: None, gas: None, ..Default::default() };
```
Comment (mattsse) [inline crates/rpc/rpc-testing-util/src/trace.rs:329]: this should just be `CallRequest::default()`

File: crates/rpc/rpc-testing-util/src/trace.rs [hunk 2]
```diff
@@ -319,4 +320,29 @@ mod tests {
         println!("Total successes: {}", successes);
         println!("Total failures: {}", failures);
     }
+
+    #[tokio::test]
```
Comment (mattsse) [inline crates/rpc/rpc-testing-util/src/trace.rs:324]: we want to ignore this for now, but is useful for manual testing
```suggestion
    #[tokio::test]
    #[ignore]
```

**Review by mattsse — CHANGES_REQUESTED** [review]

**Comment by DoTheBestToGetTheBest** [comment]
Thank you so so much @mattsse 

Does it look good for you?

**Review by mattsse — APPROVED** [review]
yes lgtm champ

**Pull request merged**